Makefile

```diff
@@ -35,9 +35,9 @@ CONTAINER_RUNTIME_COMMAND := $(or ${CONTAINER_COMMAND}, ${CONTAINER_RUNTIME_COMM
 LINT_CODE_STYLING_DIRS := src/tests src/triggers src/assisted_test_infra/test_infra src/assisted_test_infra/download_logs src/service_client src/consts src/virsh_cleanup src/cli
 
 # assisted-service
-SERVICE := $(or $(SERVICE), quay.io/edge-infrastructure/assisted-service:latest)
+SERVICE := $(or $(SERVICE), quay.io/edge-infrastructure/assisted-service:ocm-2.12)
```

Comment:
Looks like this image don't exist yet. What should be creating it?

```
$ podman pull quay.io/edge-infrastructure/assisted-service:ocm-2.12
Trying to pull quay.io/edge-infrastructure/assisted-service:ocm-2.12...
Error: initializing source docker://quay.io/edge-infrastructure/assisted-service:ocm-2.12: reading manifest ocm-2.12 in quay.io/edge-infrastructure/assisted-service: manifest unknown
```

Reply:
they should be mirrored by our nightly job, we need to wait for tomorrow to merge this PR. Thanks for checking!